docs/blockentities/container.md

```diff
@@ -254,7 +256,11 @@ public class MyBackpackContainer extends SimpleContainer {
 }
 ```
 
-And voilà, you have created an item-backed container! Simply call `new MyBackpackContainer(stack)` to create a container for a menu or other use case.
+And voilà, you have created an item-backed container! Call `new MyBackpackContainer(stack)` to create a container for a menu or other use case.
+
+:::warning
+Be aware that `Menu`s that directly interface with `Container`s must `#copy()` their `ItemStack`s when modifying them, as otherwise the immutability contract on data components is broken.
```

Comment:
This should specifically refer to `StackCopySlot` as I mentioned here: https://github.com/neoforged/Documentation/pull/160#discussion_r1774248427. There is no reason for mods to reinvent the wheel when NeoForge already provides this.